.travis.yml

```diff
@@ -3,6 +3,8 @@ language: php
 php:
   - '7.0'
   - '7.3'
+  - '7.4'
+  - nightly
```

Comment:
Is there an explicit version we could set here so that CI can't break for the next major PHP version?

Reply:
At the time when I implemeted that, Travis did not support the handle `8.0`, only when using `nightly`. But it seems that this is still the case: https://travis-ci.community/t/php-8-0-missing/10132

Reply:
With GitHub actions there is a way to do that, though.

Reply:
Alright, doesn't need to block this PR then